tests/md5/md5.cc

```diff
@@ -0,0 +1,374 @@
+/* MD5
+ converted to C++ class by Frank Thilo (thilo@unix-ag.org)
+ for bzflag (http://www.bzflag.org)
```

Comment:
the licensing of this is really unclear to me. Do you mind if I show you my suggestion?